### PR TITLE
style: make info page responsive

### PR DESCRIPTION
**Pull request by tokitou-san**
Helpppp I need helpppppppppppppp
I dont want tattoo helpp
You will get tadu

anime-cry

### CODE REVIEW

File: src/lib/components/pages/anime_info/anime_episodes.svelte [hunk 1]
```diff
@@ -18,158 +18,161 @@
...
-
-                        <span class="flex h-full place-items-center rounded-[0.25vw] bg-surface-400 px-[0.9vw] leading-[0.9vw]">1080p</span>
-
-                        <span class="flex h-full place-items-center rounded-[0.25vw] bg-surface-400 px-[0.9vw] leading-[0.9vw]">720p</span>
-                        <span class="flex h-full place-items-center rounded-[0.25vw] bg-surface-400 px-[0.9vw] leading-[0.9vw]">480p</span>
+                        {#each Array("1080p", "720p", "480p") as resolution}
+                            <span class="flex h-full place-items-center rounded bg-surface-400 px-2 leading-[0.9vw] md:rounded-[0.25vw] md:px-[0.9vw]">
+                                {resolution}
+                            </span>
+                        {/each}
```
Comment (baseplate-admin) [inline src/lib/components/pages/anime_info/anime_episodes.svelte:57]:
```svelte
  {#each ["1080p", "720p", "480p"] as resolution}
                            <span class="flex h-full place-items-center rounded bg-surface-400 px-2 leading-[0.9vw] md:rounded-[0.25vw] md:px-[0.9vw]">
                                {resolution}
                            </span>
                        {/each}
```

**Review by baseplate-admin — CHANGES_REQUESTED** [review]
Change it please

File: src/lib/components/pages/anime_info/anime_episodes.svelte [hunk 2]
```diff
@@ -18,158 +18,161 @@
...
                         </div>
-                    </div>
+                    </show-on-hover>
                 </div>
-                <span class="pt-[0.75vw] text-[0.8vw] font-light leading-[0.9375vw] text-surface-50/75 transition duration-300 group-hover:text-surface-50">
+                <span class="pt-2 text-[0.5rem] font-light leading-normal text-surface-50/75 transition duration-300 group-hover:text-surface-50 md:pt-[0.75vw] md:text-[0.8vw] md:leading-[0.9375vw]">
                     {episode.episode_name}
                 </span>
             </div>
         {/each}
```
[the remark this answers is not in the document]
Comment (baseplate-admin) [reply src/lib/components/pages/anime_info/anime_episodes.svelte:155]: image

This is how it's supposed to look like

[the remark this answers is not in the document]
Comment (baseplate-admin) [reply src/lib/components/pages/anime_info/anime_episodes.svelte:155]: image

[the remark this answers is not in the document]
Comment (baseplate-admin) [reply src/lib/components/pages/anime_info/anime_episodes.svelte:155]: Can you spot the difference ?

[the remark this answers is not in the document]
Comment (tokitou-san) [reply src/lib/components/pages/anime_info/anime_episodes.svelte:155]: > This is how it's supposed to look like

Oky image not rendered that time.

Yea I see, do we need a toggle button to change between these two layouts? or just keep that?

**Comment by baseplate-admin** [comment]
image

hey i have found a regression

**Comment by baseplate-admin** [comment]
image


Notice how the ui becomes transparent ?

**Comment by tokitou-san** [comment]
I want fishhhh

**Comment by baseplate-admin** [comment]
🗿

**Pull request merged**